.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Just trying a new (to me) github feature.
Would this be an appropriate place for those - what did you call them - friction logs ?
If so, please use it, if not, please show me how it should be done.